mimic/model/ironic_objects.py

```diff
@@ -170,6 +175,8 @@ def detail_json(self):
         if self.instance_uuid:
             template["instance_info"] = self.static_instance_info
             template["provision_state"] = "active"
+        if not self.driver:
+            template["driver"] = "agent_ipmitool"
```

Comment:
So, that was first instinct as well. But for some reason, that I cant figure why, if I did that, all the nodes seem to have self.driver set to the value sent for the first node's driver.